pastas/stressmodels.py

```diff
@@ -225,6 +225,12 @@ def get_nsplit(self):
             return self.nsplit
         else:
             return len(self.stress)
+        
+    def get_block(self, p, dt, tmin, tmax):
+        """Internal method to get the block-response from the respnse function"""
+        maxtmax = (tmax-tmin)/pd.to_timedelta(1,'d')
+        b = self.rfunc.block(p, dt, maxtmax=maxtmax)
+        return b
```

Comment:
I think it would be better to rename this to "get_block_response" or simply "block". And then maybe we refactor step to step_response as well while we are at it.

Also I think refund.block requires the cutoff?

Reply:
This is in stressmodels.py, so there is no step here. I use get_block() instead of block() to show the difference between the block-method in rfunc.py. The rfunc gets the cutoff from self, as it was passed earlier already from the stressmodel.